packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/onboarding/translations/Messages_fr_FR.json

```diff
@@ -1,9 +1,8 @@
 {
   "pci_projects_project_storages_objects_onboarding_title": "Object Storage",
-  "pci_projects_project_storages_objects_onboarding_content1": "Vous n'avez pas encore créé de service d'Object Storage.",
-  "pci_projects_project_storages_objects_onboarding_content2": "Profitez du stockage illimité à la demande, accessible par API.",
-  "pci_projects_project_storages_objects_onboarding_content3": "L’Object Storage consiste à placer vos données dans un espace illimité, accessible directement via une API HTTP. Vous pouvez ainsi intégrer l'accès à ces informations depuis vos applications, grâce aux nombreuses librairies OpenStack Swift existantes. Outre l’administration des droits d'accès, l'Object Storage intègre des fonctionnalités de « stockage intelligent » comme la gestion des versions et des dates d'expiration, ainsi que la synchronisation entre différentes localisations.",
-  "pci_projects_project_storages_objects_onboarding_content4": "Vos données sont sécurisées sur trois réplicas, afin de garantir leur durée de vie, et vous profitez d'une facturation à l'usage dès le premier Go.",
+  "pci_projects_project_storages_objects_onboarding_content1": "Déposez vos fichiers statiques (vidéos, images, fichiers web…) dans un espace illimité, pour les exploiter depuis votre application ou pour les rendre accessibles sur le web via notre API S3.",
+  "pci_projects_project_storages_objects_onboarding_content2": "Cet espace de stockage évolue en même temps que votre projet, sans que vous ayez besoin de le prévoir à l'avance.C'est l'endroit idéal pour le stockage de données exigeant en performances.",
```

Comment:
l'avance.C'est => l'avance. C'est

One space is missing.

Reply:
Okay will add it.